.gitlab/build-rocm.sh

```
@@ -0,0 +1,446 @@
+        ./contrib/configure-release-mt \
+            --prefix="${UCX_INSTALL_DIR}" \
+            --enable-shared \
+            --disable-static \
+            --disable-doxygen-doc \
+            --enable-optimizations \
+            --enable-cma \
+            --enable-devel-headers \
+            --with-verbs \
+            --with-dm \
+            --without-gdrcopy \
+            ${UCX_CUDA_BUILD_ARGS} && \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)